s3cmd

```diff
@@ -470,7 +470,7 @@ def cmd_object_put(args):
             ret = EX_PARTIAL
             continue
         except InvalidFileError as exc:
-            error(u"Upload of '%s' is not possible (Reason: %s)" % (full_name_orig, exc))
+            error(u"Upload of '%s' is not possible. Reason: %s." % (full_name_orig, exc))
```

Comment:
In all other error messages the reason section is in parentheses, I think this line should remain unchanged.